test/presenters/notifications/setup/review.presenter.test.js

```diff
@@ -26,8 +27,12 @@ describe('Notifications Setup - Review presenter', () => {
     }
 
     testRecipients = RecipientsFixture.recipients()
+    // This data is used to ensure the recipients are grouped when they have the same licence ref / name.
```

Comment:
Just want to highlight if this is acceptable. 

We could manually add them in rather than use the fixture (or use the fixtures as a base and create them in this test to have exact message_types etc)

happy to change as i think this is becoming a bit unwieldy to use now we introduce the sort / duplicates.